public/js/update-experimenter.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
One thing to add is the current user profile data, so that we don't land into a page with a blank form